public/app/features/plugins/extensions/registry/setup.ts

```diff
@@ -1,3 +1,6 @@
+import CentralAlertHistorySceneExposedComponent from 'app/features/alerting/unified/components/rules/central-state-history/CentralAlertHistorySceneExposedComponent';
+
+import { PluginExtensionExposedComponents } from '../../../../../../packages/grafana-data/src/types/pluginExtensions';
```

Comment:
nit: `PluginExtensionExposedComponents` needs to be exposed from `@grafana/data` so it can be imported without using relative path imports.